insights/combiners/tests/test_httpd_conf_tree.py

```diff
@@ -522,3 +523,16 @@ def test_httpd_conf_jbcs_httpd24_tree():
     assert len(load_module_list) == 4
     assert result['LoadModule'][3].value == 'mpm_prefork_module modules/mod_mpm_prefork.so'
     assert result['LoadModule'][3].file_path == '/opt/rh/jbcs-httpd24/root/etc/httpd/conf.modules.d/02-c.conf'
+
+
+def test_httpd_nested_conf_file():
+    httpd1 = _HttpdConf(context_wrap(HTTPD_CONF_MAIN_3, path='/etc/httpd/conf/httpd.conf'))
+    httpd2 = _HttpdConf(context_wrap(HTTPD_CONF_FILE_1, path='/etc/httpd/conf.d/00-a.conf'))
+    httpd3 = _HttpdConf(context_wrap(HTTPD_CONF_FILE_2, path='/etc/httpd/conf.d/d1/hello.conf'))
+    result = HttpdConfTree([httpd1, httpd2, httpd3])
+
+    server_root = result['ServerRoot'][last]
```

Comment:
The `last` works well, but to keep consistent with the upcoming #1975, better modify it to `-1`.